libgalois/src/PropertyGraph.cpp

```diff
@@ -500,7 +608,7 @@ katana::PropertyGraph::ConstructEntityTypeIDs() {
   uint64_t num_node_rows = static_cast<uint64_t>(node_properties()->num_rows());
   if (num_node_rows == 0) {
     node_entity_type_id_ = GetUnknownEntityTypeIDs(num_nodes());
-  } else {
+  } else {  ///TODO: Giorgi <---------------Not sure if this is the right way to parameterize the call for the GetEntityTypeIDsFromProperties()
```

Comment:
Not sure why you need to modify this function as this is for when storage doesn't have type info.